target/linux/mediatek/image/mt7623.mk

```diff
@@ -1,5 +1,9 @@
 KERNEL_LOADADDR := 0x80008000
 
+FEATURES += ext4 usb
```

Comment:
Are you sure? Usually FEATURES is in target/subtarget Makefile, not image Makefile.
And AFAIK DEVICE_PACKAGES is a device-specific variable so it should be defined in device recipes.

Reply:
DEVICE_PACKAGES is device-specific, but defining it like below still works if none of the devices has it defined (which seems to be the case here).